src/types.js

```diff
@@ -14,27 +14,39 @@
  * @property {string} [msgctxt] Context of the message.
  * @property {string} msgid The singular message ID.
  * @property {string} [msgid_plural] The plural message ID.
- * @property {string} msgstr Array of translated strings.
+ * @property {string[]} msgstr Array of translated strings.
  * @property {GetTextComment} [comments] Comments associated with the translation.
- * @property {GetTextTranslation} [obsolete] Whether the translation is obsolete.
+ * @property {boolean} [obsolete] Whether the translation is obsolete.
+ */
+
+/**
+ * @typedef {Record<string, Record<string, GetTextTranslation>>} Translations The translations index.
  */
 
 /**
  * Represents GetText translations.
  * @typedef {Object} GetTextTranslations
  * @property {string|undefined} charset Character set.
- * @property {{[key: string]: string}} headers Headers.
- * @property {GetTextTranslations['translations']} [obsolete] Obsolete messages.
- * @property {Object.<[msgctxt: string], Object.<[msgid: string], GetTextTranslation>>} translations Translations.
```

Comment:
I didn't understand this type, it was coming up as `any`. Though I typed the `Translations` and used it.

Reply:
if can help, should type this

```
  "msgctxt": {
      "msgid": {
          "msgid": "",
          "msgstr": [
              "test"
          ]
      },
      "2": {
          "2": "",
          "msgstr": [
              "test"
          ]
      }
  }
```

Reply:
That what I typed it as, I got it from the docs. 

What I meant, I didn't understand the original type, did it work for you?